include/nds/system.h

```diff
@@ -457,11 +457,47 @@ void resetARM9(u32 address);
 
 // DSi SCFG registers
 
-#define REG_SCFG_ROM            *(vu16 *)0x4004000
-#define REG_SCFG_CLK            *(vu16 *)0x4004004
-#define REG_SCFG_RST            *(vu16 *)0x4004006
-#define REG_SCFG_EXT            *(vu32 *)0x4004008
-#define REG_SCFG_MC             *(vu16 *)0x4004010
+#define REG_SCFG_ROM            (*(vu16 *)0x4004000)
+
+#define REG_SCFG_CLK            (*(vu16 *)0x4004004)
+
+#define SCFG_CLK_ARM9_134       (1 << 0)
```

Comment:
These defines are largely duplicates.

See:

```c
#ifdef ARM9
#define SCFG_CLK_ARM9_TWL       BIT(0)
#define SCFG_CLK_DSP            BIT(1)
#define SCFG_CLK_CAMERA_IF      BIT(2)
#define SCFG_CLK_MBK_RAM        BIT(7)
#define SCFG_CLK_CAMERA_EXT     BIT(8)
// ...
```

Reply:
Ah, I didn't notice it, too many definitions to track across different files. I've reorganized them a bit so that they are closer to their registers, and removed the duplicates.